File: src/main/java/com/lanka_guide/tripplanner/model/activity/transport/method/drive/Drive.java
package com.lanka_guide.tripplanner.model.activity.transport.method.drive;

import com.lanka_guide.tripplanner.model.activity.transport.method.TransportMethod;

import java.time.ZonedDateTime;

public class Drive extends TransportMethod {
    public Drive(ZonedDateTime from, ZonedDateTime to) {
        super(from, to);
    }
}

File: src/main/kotlin/com/lankaguide/tripplanner/model/activity/transport/drive/Drive.kt
package com.lankaguide.tripplanner.model.activity.transport.drive

import com.lankaguide.tripplanner.model.activity.transport.Transport
import com.lankaguide.tripplanner.model.place.Place

import java.time.Duration
import java.time.ZonedDateTime

abstract class Drive(override val from: Place, override val to: Place, override val start: ZonedDateTime, override val end: ZonedDateTime)
    : Transport(from, to, start, end)
